bundles/nl.esi.comma.expressions/src/nl/esi/comma/expressions/validation/ExpressionFunction.java

```diff
@@ -320,12 +321,123 @@ public Pair<Integer, String> validate(List<Expression> args) {
 		public String getDocumentation() {
 			return String.format("<T> %s(vector<T>, int, T): vector<T>", name());
 		}
+	},
+	intToString {
```

Comment:
I suggest to call this function toString. We may want to support other types in the future, or @dbera do we want to support reals now already?

```suggestion
	toString {
```

Reply:
> I suggest to call this function toString, do not forget to update it at all used places. We may want to support other types in the future, or @dbera do we want to support reals now already?
>> It might be good idea to take it along, else i am afraid this comes back very soon again.